src/3_branch.js

```diff
@@ -446,82 +446,84 @@ Branch.prototype['init'] = wrap(
 				link_identifier === sessionData['click_id'])) {
 			attachVisibilityEvent();
 			checkHasApp(sessionData, finishInit);
+			return;
 		}
-		else {
-			if (CORDOVA_BUILD || TITANIUM_BUILD) {
 
-				var apiCordovaTitanium = function(data) {
-					if (!freshInstall) {
-						data['identity_id'] = sessionData['identity_id'];
-						data['device_fingerprint_id'] = sessionData['device_fingerprint_id'];
+		if (WEB_BUILD) {
```

Comment:
i'm pretty sure we still need an else, no? we can definitely turn it into an elseif though